### PR TITLE
Move defense_component to portfolios model

### PR DESCRIPTION
**Pull request by montana-mil**
## Description
`defense_component` is now an attribute on the `Portfolio` model instead of `TaskOrder`. If a new task order is created outside the context of a portfolio, it creates a portfolio that has the selected `defense_component`. If the task order is created from within the context of an existing portfolio, the defense component of the task order is that of its portfolio.

The defense component is displayed (read only) on the Portfolio Settings (admin) screen.

## Pivotal Tracker
https://www.pivotaltracker.com/story/show/163954462

## Screenshots
screen shot 2019-02-26 at 11 35 19 am

### CODE REVIEW

File: templates/fragments/task_order_review/app_info.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <div class="row">
   {{ ReviewField(("task_orders.new.review.portfolio" | translate), task_order.portfolio_name) }}
-  {{ ReviewField(("task_orders.new.review.dod" | translate), task_order.defense_component, filter="normalizeOrder") }}
+  {{ ReviewField(("task_orders.new.review.dod" | translate), task_order.portfolio.defense_component, filter="normalizeOrder") }}
```
Comment (dandds) [inline templates/fragments/task_order_review/app_info.html]: You could just use the property on `task_order`, right?

File: templates/portfolios/admin.html [hunk 2]
```diff
@@ -24,6 +24,12 @@
           <button type="submit" class="usa-button usa-button-big usa-button-primary" tabindex="0">Save</button>
         </div>
       </div>
+      <div class='defense-row'>
```
Comment (dandds) [inline templates/portfolios/admin.html]: lol this class name

File: tests/routes/task_orders/test_new_task_order.py [hunk 3]
```diff
@@ -203,7 +209,8 @@ def test_show_task_order_form(task_order):
         task_order.creator, task_order_id=task_order.id
     )
     assert (
-        another_workflow.form.data["defense_component"] == task_order.defense_component
```
Comment (dandds) [inline tests/routes/task_orders/test_new_task_order.py]: This should also still work with your property on the TO model, right?

File: templates/fragments/task_order_review/app_info.html (hunk 1 above)
Comment (montana-mil) [reply templates/fragments/task_order_review/app_info.html]: oh yep! this was before i added that prop. thanks for catching

**Pull request merged**